libraries/AP_Frsky_Telem/AP_Frsky_Telem.h

```diff
@@ -127,7 +127,7 @@ class AP_Frsky_Telem
     void update_control_mode(uint8_t mode) { _ap.control_mode = mode; }
 
     // update whether we're flying (used for Plane)
-    void set_is_flying(bool is_flying) { *_ap.value == is_flying ? (*_ap.value | AP_ISFLYING_FLAG) : (*_ap.value & (~AP_ISFLYING_FLAG)); }
+    void set_is_flying(bool is_flying) { _ap.value == is_flying ? (_ap.value | AP_ISFLYING_FLAG) : (_ap.value & (~AP_ISFLYING_FLAG)); }
```

Comment:
Then the method would have no use, because it is only used for Plane and Plane doesn't construct the library with an ap.value pointer. Although this is a mess, I think other ways would be even worse.

But that `_ap.value == is_flying` looks wrong, it should just be `is_flying`.